cognee/modules/users/authentication/default/default_jwt_strategy.py

```
@@ -0,0 +1,23 @@
+import jwt
```

Ruff (0.11.9)

`jwt` imported but unused

Remove unused import: `jwt`

(F401)